src/Microsoft.Health.Dicom.Core/Features/Security/DataActions.cs

```diff
@@ -13,10 +13,16 @@ namespace Microsoft.Health.Dicom.Core.Features.Security
     public enum DataActions : ulong
 #pragma warning restore CA1028 // Enum Storage should be Int32
     {
+        [EnumMember(Value = "none")]
```

Comment:
These values are the defaults, aren't they? Although I suppose casing is property of some serializers settings, as most things in .NET will be PascalCase

Reply:
Oh, I see where this comes into play in *DicomRoleLoader.cs*. We could also assume the name of the action is the name of the value unless `EnumMember` is explicitly annotated, right?

Reply:
The casing matters, but generally yes they're basically the same. The actual value that's allowed in the json is from the `roles.schema.json`.